src/particles/distribution/Kurth6D.H

```diff
@@ -0,0 +1,141 @@
+/* Copyright 2022 Chad Mitchell, Axel Huebl
+ *
+ * This file is part of ImpactX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_DISTRIBUTION_KURTH6D
+#define IMPACTX_DISTRIBUTION_KURTH6D
+
+#include <AMReX_Random.H>
+#include <AMReX_REAL.H>
+
+#include <cmath>
+
+
+namespace impactx
+{
+namespace distribution
+{
+    struct Kurth6D
+    {
+        /** A 6D Kurth distribution
+         *
+         * Return sampling from a 6D Kurth distribution.
+         *
+         * @param sigx,sigy,sigt for zero correlation, these are the related
+         *                                   RMS sizes (length); ...
```

Comment:
leftover `...`?

Reply:
This comment was copied/pasted from the Waterbag documentation.  It is equally valid here.  I think "length" can be replaced by "m".